src/pages/Checkout/CheckoutBetaPage.vue

```diff
@@ -55,6 +55,7 @@ import LoginForm from '@/components/Forms/LoginForm';
 import RegisterForm from '@/components/Forms/RegisterForm';
 import FacebookLoginRegister from '@/components/Forms/FacebookLoginRegister';
 import BasketItemsList from '@/components/Checkout/BasketItemsList';
+import loanCardQuery from '@/graphql/query/loanCardData.graphql';
```

Comment:
Don't think we need this. Any query adjustments can be made to initializeCheckout which I think you've done already.